api/odl2.py

```diff
@@ -16,18 +16,13 @@
     ExternalIntegration,
     HasExternalIntegration,
 )
-from core.opds2_import import (
-    OPDS2Importer,
-    OPDS2ImporterConfiguration,
-    OPDS2ImportMonitor,
-    RWPMManifestParser,
-)
-from core.opds_import import ConnectionConfiguration
+from core.opds2_import import OPDS2Importer, OPDS2ImportMonitor, RWPMManifestParser
+from core.opds_import import OPDSImporterConfiguration
 from core.util import first_or_default
 from core.util.datetime_helpers import to_utc
 
 
-class ODL2APIConfiguration(OPDS2ImporterConfiguration, ConnectionConfiguration):
+class ODL2APIConfiguration(OPDSImporterConfiguration):
```

Comment:
This change looks a little suspect, but I don't understand the whats actually going on here

Reply:
Currently trying to get the IDE to show me `git blame` information. I don't _think_ that change was one of mine, but it seems harmless. There seems to have been a general move to unify OPDS and OPDS2 configuration information, and ODL is compatible with both OPDS 1.2 and OPDS 2.0.

Reply:
Don't we lose connection retry counts without ConnectionConfiguration?

Reply:
Ok I see this is added to OPDSImporter